Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -159,8 +240,9 @@ private void OnDestroy()
         Time.timeScale = 1;
         _playerTakeDamageButton.onClick.RemoveAllListeners();
         _toggleGodMode.GetComponent<Button>().onClick.RemoveAllListeners();
+        _toggleFreeLookCamButton.GetComponent<Button>().onClick.RemoveAllListeners();
```

Comment:
If you're going to serialize these objects and they arent necessary for the game, please add a null check to prevent errors before `_freeLookCam` and `_toggleFreeLookCamButton`